From 9b2143ed91f74e6b44bbb95fcce1586363351042 Mon Sep 17 00:00:00 2001
From: Jianzheng Zhou <jianzheng.zhou@freescale.com>
Date: Wed, 7 May 2014 18:01:52 +0800
Subject: [PATCH 316/602] ENGR00305958 Ethernet:change custom ethernet
 interface for sabresd_sx

There are two interfaces in sabresd_sx. eth1 is defined to be used as input
one in driver.Just change dhcod service to align with it.

Signed-off-by: Jianzheng Zhou <jianzheng.zhou@freescale.com>
---
 sabresd_6sx/init.rc | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/sabresd_6sx/init.rc b/sabresd_6sx/init.rc
index aefbe7c..ade65f3 100755
--- a/sabresd_6sx/init.rc
+++ b/sabresd_6sx/init.rc
@@ -174,6 +174,11 @@ service ing /system/bin/ingsvcd -c /system/etc/gps/Orion.ini
 service charger /charger
     class charger
 
+service dhcpcd_eth1 /system/bin/dhcpcd -ABKL
+    class main
+    disabled
+    oneshot
+
 on fs
 # mount ext4 partitions
     mount_all /fstab.freescale
-- 
1.8.0

